app/os_android.go

```diff
@@ -203,6 +203,12 @@ type ViewEvent struct {
 	// the next ViewEvent is received.
 	// A zero View means that there is currently no view attached.
 	View uintptr
+
+	// Layer is a JNI global reference to the android.widget.FrameLayout
+	// instance which holds View and others views. The reference is
+	// valid until the next ViewEvent is received.
+	// A zero Layer means that there is currently no layer attached.
+	Layer uintptr
```

Comment:
Why is https://developer.android.com/reference/android/view/View#getParent() not enpough for clients?

Reply:
I add it to make easier. If you need to provide one View to an third-party library, you can use `Layer` directly.  Otherwise, you need to write your own Java code to call `getParent()` and then provide it to the another library.

Reply:
How can you pass a View to the third-party without writing Java code?